src/parser.rs

```diff
@@ -21,6 +21,40 @@ pub(crate) struct Comments {
     pub ignore: Vec<Condition>,
     /// Only run this test if all of these filters apply
     pub only: Vec<Condition>,
```

Comment:
There's no fundamental reason these couldn't also be revisioned, is there?

Reply:
Yea, I thought for a moment they couldn't be checked, but I think we have all the information available